src/cdfDatasource.ts

```diff
@@ -63,8 +62,8 @@ export function formQueryForItems(
   };
 }
 
-function calculateDPLimitPerQuery(hasAggregates: boolean, queriesNumber: number) {
```

Comment:
for regular ts limit is still different for aggregated/not aggregated dps

Reply:
ah, right, nice catch!